concepts/05 UI Components/List/00 Getting Started with List/04 Select Items.md

```diff
@@ -0,0 +1,14 @@
+The **List** supports the following selection modes: **single**, **multiple**, and **all**. They can be specified in the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property. End users click or tap **List** items to select them. Selected items become shaded. When the [showSelectionControls](/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls) property is set to **true**, users can select items with radio buttons or checkboxes.
```

Comment:
I know I requested that you added the information about how users select items, but on the second glance it doesn't quite fit the narrative.

```suggestion
The **List** supports the following item selection modes: *"single"*, *"multiple"*, and *"all"*. To enable selection, specify one of these modes in the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property.
```